Enterprise-CMCS/macpro-mdct-mcr
# statusing FE

## Summary
Created the front-end for the status summary view. 

### Description


### Related ticket

https://qmacbis.atlassian.net/browse/MDCT-2051

### How to test

Create programs at different stages of completeness and view the results page.

If there are any error or the program is in the "Not started" status, the user should not be able to submit the form and an error message should appear.

### Important updates


### Author checklist
